clicc_devices/fixtures/sets.json

```diff
@@ -0,0 +1,68 @@
+[
+{
+  "model": "clicc_devices.set",
+  "pk": 3,
```

Comment:
The `pk` values in the fixture should be 1-6.  The difference here probably is due to sets being created manually via Admin earlier, then deleted.

It's not critical in dev, (and TBH, not in production) but the eventual production sets will be created by manually applying this fixture, and we'll expect to start from 1 there.